<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QHBoxLayout" name="HLresultados" stretch="0,0,0">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
             <horstretch>39</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>278</width>
             <height>84</height>
            </size>
           </property>
           <property name="title">
            <string>Personalizar vista</string>
           </property>
           <widget class="QWidget" name="horizontalLayoutWidget_5">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>20</y>
              <width>261</width>
              <height>24</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_16">
             <item>
              <widget class="QRadioButton" name="rbCantClusters">
               <property name="sizePolicy">
                <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Cantidad Clusters</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinCantClusters">
               <property name="maximumSize">
                <size>
                 <width>50</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>100000</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="horizontalLayoutWidget_6">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>50</y>
              <width>266</width>
              <height>24</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_17">
             <item>
              <widget class="QRadioButton" name="rbElemPorCluster">
               <property name="sizePolicy">
                <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Cantidad Elementos por Cluster</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinCantElemPorCluster">
               <property name="maximumSize">
                <size>
                 <width>50</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>100000</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>220</width>
             <height>80</height>
            </size>
           </property>
           <property name="title">
            <string>Métricas</string>
           </property>
           <widget class="QWidget" name="horizontalLayoutWidget_4">
            <property name="geometry">
             <rect>
              <x>60</x>
              <y>10</y>
              <width>121</width>
              <height>31</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_15">
             <item>
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>Método 1</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPlainTextEdit" name="txtMet1">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="maximumSize">
                <size>
                 <width>50</width>
                 <height>28</height>
                </size>
               </property>
               <property name="plainText">
                <string>20</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="horizontalLayoutWidget_3">
            <property name="geometry">
             <rect>
              <x>60</x>
              <y>40</y>
              <width>121</width>
              <height>31</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_18">
             <item>
              <widget class="QLabel" name="label_15">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>28</height>
                </size>
               </property>
               <property name="text">
                <string>Método 2</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPlainTextEdit" name="txtMet2">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="maximumSize">
                <size>
                 <width>50</width>
                 <height>28</height>
                </size>
               </property>
               <property name="plainText">
                <string>35</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>288</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="VLayoutArbol" stretch="0">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QVBoxLayout" name="SL">
           <item>
            <widget class="QLabel" name="txtSL">
             <property name="text">
              <string>Single-Link</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="SL2">
             <item>
              <layout class="QVBoxLayout" name="VLgraficoDendogramaSL">
               <property name="sizeConstraint">
                <enum>QLayout::SetMaximumSize</enum>
               </property>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="VLgraficoSL"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="CL">
           <item>
            <widget class="QLabel" name="txtCL">
             <property name="text">
              <string>Complete-Link</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="CL2">
             <item>
              <layout class="QVBoxLayout" name="VLgraficoDendogramaCL">
               <property name="sizeConstraint">
                <enum>QLayout::SetMaximumSize</enum>
               </property>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="VLgraficoCL">
               <property name="sizeConstraint">
                <enum>QLayout::SetMaximumSize</enum>
               </property>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="AL">
           <item>
            <widget class="QLabel" name="txtAL">
             <property name="text">
              <string>Average-Link</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="AL2">
             <item>
              <layout class="QVBoxLayout" name="VLgraficoDendogramaAL">
               <property name="sizeConstraint">
                <enum>QLayout::SetMaximumSize</enum>
               </property>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="VLgraficoAL">
               <property name="sizeConstraint">
                <enum>QLayout::SetMaximumSize</enum>
               </property>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_12" stretch="">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
